clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -598,12 +608,30 @@ message SearchResponse {
                 google.protobuf.Struct structValue = 8; // Value for structured data
                 // Value for VECTOR FieldType
                 Vector vectorValue = 9;
+                RepeatedFieldValues repeatedFieldValues = 10;
```

Comment:
Would it be easier to use the existing `ListValue` type https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/struct.proto#L92

Reply:
Thanks for the pointer. I spent a lot of time creating this and its builder methods. Using `ListValue` and the existing struct builders would have saved me a lot of time, if I had seen them earlier.